rke/k8s_rke_system_images.go

```diff
@@ -8990,6 +8990,47 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		"v1.24.17-rancher1-2": {
```

Comment:
@jeffinkottaram Could you update `v1.24.17-rancher1-1` instead of introducing a new `rancher1-2`? We added v1.24.17 recently, which hasn't been released yet so we can bump ACI versions for the same version.